rest-api/src/main/resources/paths/studies/v3_studies_studyId_participants_userId_studyStartDate.yml

```diff
@@ -0,0 +1,19 @@
+get:
+    operationId: getStudyStartDate
+    summary: Get the participant's start date in study
+    tags:
+        - _For Workers    
+    security:
+        - BridgeSecurity: []
+    parameters:
+        - $ref: ../../index.yml#/parameters/studyId
+        - $ref: ../../index.yml#/parameters/userId
+    responses:
+        200:
+            description: OK
+            schema:
+                $ref: ../../definitions/date_time_holder.yml
+        401:
+            $ref: ../../responses/401.yml
+        403:
+            $ref: ../../responses/403_not_worker.yml
```

Comment:
Let's you, me, and Dwayne discuss this. This is too fine-grained for my taste. I think this event should be calculated and added to the events mapping you can already retrieve for any user from the API. This then means you don't have to add an entire API just to add this event.

Reply:
We're going to move STUDY_START_DATE into the map of activity events created for every user. That way it can be retrieved through existing APIs, *and* it's visible what the calculation is to end users and we can schedule against it if we wish.

Reply:
Sounds good to me, but we should do it in a backwards compatible way. I don't think we want to have to backfill this for all users.